test/test.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const parser = require('./../src/parser.js');
+const chai = require('chai');
+const docDefault = fs.readFileSync('./test/config/doc.txt', 'utf8');
+const configDefault = require('./../src/template/configDefault.json');
+const mocha = require('mocha');
+const assert = require('assert');
+
+const main = (testsData) => {
+    testsData.tests.forEach((test) => {
+        mocha.it(`${test.description}`, (done) => {
+            const config = test.config ? require(test.config) : configDefault;
+            const doc = test.doc ?
+                fs.readFileSync(test.doc, 'utf8') : docDefault;
+            runTest(test, doc, config, done);
+        });
+    });
+};
+
+const runTest = (test, doc, config, done) => {
```

Comment:
Please, add JSDoc

Reply:
@svoboda-rabstvo it's another task, maybe it is better to add JSDoc to PR #123 after this PR being merged?